manifests/networktopology/deploy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: network-aware-scheduler
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      component: scheduler
+      tier: control-plane
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      serviceAccountName: network-aware-scheduler
```

Comment:
The following rules have to be added into the ClusterRole, which is bound to the ServiceAccount with name `network-aware-scheduler`:
```
- apiGroups: ["appgroup.diktyo.k8s.io"]
  resources: ["appgroups"]
  verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
- apiGroups: ["networktopology.diktyo.k8s.io"]
  resources: ["networktopologies"]
  verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
```
Probably RBAC yaml file containing the ServiceAccount, ClusterRole, ClusterRoleBinding, Role and RoleBinding can be added inside `manifests/networktopology/` directory, similar to this one - https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/manifests/noderesourcetopology/cluster-role.yaml?